src/cartesia/tts/_async_websocket.py

```
@@ -453,8 +461,13 @@ async def _process_responses(self):
                 flush_id = response.get("flush_id", -1)
                 if context_id in self._context_queues:
                     await self._context_queues[context_id][flush_id].put(response)
-        except Exception as e:
```

Shouldn't we still deal with general exceptions? Why remove this?

---

The previous logic was catching all exceptions, setting `_error`, and then re-raising them. Now that we've gotten rid of `_error`, there's no point in catching all exceptions just to re-raise them (we should just let them bubble implicitly). The only errors that should not be raised are errors reading from a closed websocket, which are all of type `WSMessageTypeError`.